pydm/widgets/analog_indicator.py

```diff
@@ -0,0 +1,1576 @@
+from PyQt5.QtCore import QPointF
+
+from .base import PyDMWidget, TextFormatter
+from qtpy.QtGui import QColor, QPolygon, QPen, QPainter
+from qtpy.QtWidgets import QFrame, QVBoxLayout, QHBoxLayout, QLabel, QSizePolicy, QWidget, QGridLayout
+from qtpy.QtCore import Qt, QPoint, Property
```

Comment:
It's usually best practice to import from `qtpy` instead of directly from `PyQt5`. This will redirect the import to `PyQt5` if that's what the user has installed, but will also help with cross-support to making your widget work in `PySide2` and other qt python bindings.

```suggestion
from .base import PyDMWidget, TextFormatter
from qtpy.QtGui import QColor, QPolygon, QPen, QPainter
from qtpy.QtWidgets import QFrame, QVBoxLayout, QHBoxLayout, QLabel, QSizePolicy, QWidget, QGridLayout
from qtpy.QtCore import Qt, QPoint, QPointF, Property
```